aten/src/ATen/native/Embedding.cpp

```diff
@@ -144,6 +144,8 @@ Tensor & embedding_renorm_cpu_(
   auto sorted_indices = std::vector<int64_t>(data_ptr, data_ptr + num_indices);
   std::sort(sorted_indices.begin(), sorted_indices.end(), std::less<int64_t>());
 
+  // Note that we cannot use at::parallel_for here because we perform operations
+  // on Tensor inside the loop. See #28370 for more details.
```

Comment:
nit: use the full link.  It's not obvious that this is an issue or a PR, and it's not obvious we'll use github forever.